toolkit/tools/imagepkgfetcher/imagepkgfetcher.go

```diff
@@ -38,6 +38,7 @@ var (
 	usePreviewRepo       = app.Flag("use-preview-repo", "Pull packages from the upstream preview repo").Bool()
 	disableDefaultRepos  = app.Flag("disable-default-repos", "Disable pulling packages from PMC repos").Bool()
 	disableUpstreamRepos = app.Flag("disable-upstream-repos", "Disables pulling packages from upstream repos").Bool()
+	repoSnapshotTime     = app.Flag("repo-snapshot-time", "Repo time limit for tdnf virtual snapshot").String()
```

Comment:
we should clarify this is optional?  Is it optional?

Reply:
Yes the flag is optional, I will add a note in the description.